OffchainLabs/retryable-tx-panel
# refactor: add wagmi

### Summary

Replace `useWallet` library with `wagmi`

### Steps to test

- Create a retryable transaction (tutorial [here](https://github.com/OffchainLabs/arbitrum-tutorials/tree/master/packages/redeem-failed-retryable))
- Past the transaction hash
- You should be able to connect your wallet
- Redeem button should be displayed
- On click, page should reload: transaction is successful
